src/main/java/teammates/storage/sqlentity/Student.java

```diff
@@ -0,0 +1,156 @@
+package teammates.storage.sqlentity;
+
+import java.time.Instant;
+import java.util.Objects;
+
+import org.hibernate.annotations.CreationTimestamp;
+import org.hibernate.annotations.UpdateTimestamp;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.OneToOne;
+import jakarta.persistence.Table;
+
+/**
+ * Represents a Student entity.
+ */
+@Entity
+@Table(name = "Students")
+public class Student extends User {
+    // Cascade?
+    // from hibernate docs: seems like we can omit this
+    // https://docs.jboss.org/hibernate/orm/6.1/userguide/html_single/Hibernate_User_Guide.html#entity-inheritance-joined-table
+    @OneToOne(mappedBy = "id")
+    @Column(nullable = false)
+    private int id;
+
+    // Cascade?
+    @OneToOne
+    @JoinColumn(name = "teamId")
+    private Team team;
+
+    @Column(nullable = false)
+    private String comments;
+
+    @CreationTimestamp
+    @Column(nullable = false, updatable = false)
+    private Instant createdAt;
+
+    @UpdateTimestamp
+    @Column(nullable = false)
+    private Instant updatedAt;
+
+    protected Student() {
+        // required by Hibernate
+    }
+
+    public Student(StudentBuilder builder) {
+        this.setId(builder.id);
+        this.setTeam(builder.team);
```

Comment:
Let's comment out all team/account related code. Need to be able to build the project.